modules/dnn/test/test_model.cpp

```diff
@@ -273,7 +273,7 @@ TEST_P(Test_Model, Keypoints_pose)
 #endif
 
     Mat inp = imread(_tf("pose.png"));
-    std::string weights = _tf("onnx/models/lightweight_pose_estimation.onnx", false);
+    std::string weights = _tf("onnx/models/lightweight_pose_estimation_201912.onnx", false);
```

Comment:
So may I ask you to highlight once again the difference comparing previous version?